### ISSUE TITLE
[feat] More output on the build step

### ISSUE THREAD

**Issue by markusthoemmes** [boson-project/functions#18]
**Describe the solution you'd like**
My `kn faas build` command was literally

```console
$ kn faas build
Building image: docker.io/markusthoemmes/node:latest
```

A spinner would've been neat as it has been sitting there for a while. A `Success` message would also be nice on... well... success.

**Comment by zroubalik** [comment]
btw you can run 
```
$ kn faas build -v (--verbose)
```
to see the actual build output

**Comment by markusthoemmes** [comment]
A spinner for the general case would still be nice. The same is true for `kn faas deploy`. A `Success` message was kinda lacking there too.

**Comment by nainaz** [comment]
Agreed, a spinner and success message is needed. especially the success message.
